presto-docker/bin/entrypoint

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash -xeu
+
+# Copy config files, if they exist, to /usr/lib/presto/etc. This is necessary
+# for Kubernetes as Secrets and ConfigMaps are mounted read only.
+if [[ -d /usr/lib/presto/conf ]]; then
```

Comment:
The `find ... -delete` serves this purpose. Additionally I don't want to strongly couple the configuration with the catalogs which I plan to mount separately, hence the slight complexity here.

Reply:
> since the user may not want the default jmx/memory/tpch catalog to continue exist.

is that ok not keep these 3 default catalogs when building a presto container image?

Reply:
Effectively if you want those catalogs you can configure them yourself by mounting in the appropriate catalogs. We provide default catalogs so that a user can do a quick start of this image and play around with Presto. Additionally it presents a problem for administrators who want to limit which catalogs are available to their users.